### PR TITLE
SR2024 - Update Section - Motor board API

### PR DESCRIPTION
**Pull request by 8BitJosh**
- Update with new features
- Increase clarity

### CODE REVIEW

File: programming/motors.md [hunk 1]
```diff
@@ -6,77 +6,82 @@ title: Motor Board API
 Motor Board API
 ===============
 
-The `motor_board` object is used to control a collection of Motor Boards.
+The kit can control multiple motors simultaneously
```
Comment (RealOrangeOne) [inline programming/motors.md:9]:
```suggestion
The kit can control multiple motors simultaneously.
```

File: programming/motors.md [hunk 2]
```diff
@@ -6,77 +6,82 @@ title: Motor Board API
...
+
+my_motor_board = robot.motor_boards["srABC1"]
+my_other_motor_board = robot.motor_boards["srXYZ1"]
 ~~~~~
 
 <div class="warning" markdown="1">
-  When you have more than one Motor board connected to your kit,
-  you must use `R.motor_boards` and index by serial number. This is so
-  that the kit knows which Motor Board you want to control.
+  When you have more than one Motor Board connected to your kit, you must use `robot.motor_boards` and index by serial number.
```
Comment (RealOrangeOne) [inline programming/motors.md:47]: Nit: Be specific that the issue is not using `robot.motor_board` vs needing to use `robot.motor_boards`.

```suggestion
  When you have more than one Motor Board connected to your kit, you can't use `robot.motor_board`.
```

File: programming/motors.md [hunk 3]
```diff
@@ -85,63 +90,68 @@ motors at once.
...
+robot.motor_boards["srABC1"].motors[0].power = 0
+robot.motor_boards["srABC1"].motors[1].power = 0
 ~~~~~
 
 <div class="info" markdown="1">
-  You will need to work out for your robot which values (positive or negative)
-  result in it moving in each direction. This will depend on how you have
-  positioned your motors as well as how they have been wired to the motor board.
+  You will need to work out for your robot which values (positive or negative) result in it moving in each direction.
+  This will depend on how you have positioned your motors as well as how they have been wired to the Motor Board.
```
Comment (RealOrangeOne) [inline programming/motors.md:106]: Suggestion: Perhaps a tip here that it's likely easier to swap the wiring than having motors spin in different directions, and that it's easier for humans to have positive values be forward and negative backwards.

File: programming/motors.md (hunk 1 above)
Comment (8BitJosh) [reply programming/motors.md]: a40db43

File: programming/motors.md (hunk 2 above)
Comment (8BitJosh) [reply programming/motors.md]: a40db43

File: programming/motors.md (hunk 3 above)
Comment (8BitJosh) [reply programming/motors.md]: a40db43

File: programming/motors.md [hunk 4]
```diff
@@ -6,77 +6,82 @@ title: Motor Board API
...
-The serial number of each detected Motor Board is printed to the log when your robot starts.
+If you have more than one Motor Board attached, you need to specify which one you want to control.
+This is done using the serial number of the board.
+
+The serial number is physically written on each board and the serial of any connected board will be printed to your log when the robot starts.
 It will look something like this:
 
 ~~~~~ not-code
-sr.robot3.robot INFO - Found Student Robotics Motor Board v4 - srABC1
+sr.robot3.robot - INFO - Found MotorBoard, serial: srABC1
```
Comment (WillB97) [inline programming/motors.md:33]: Do we want log lines from both the boards below to be consistent?

Comment (8BitJosh) [reply programming/motors.md:33]: Good idea - 61ebb2c

**Review by WillB97 — APPROVED** [review]

**Pull request merged**